Validation/Geometry/test/runP_InnerServices_cfg.py

```diff
@@ -6,9 +6,10 @@
 
 #Geometry
 #
-process.load("Geometry.CMSCommonData.cmsSimIdealGeometryXML_cfi")
-
+process.load("Geometry.CMSCommonData.cmsExtendedGeometry2017XML_cfi")
 process.load("Geometry.TrackerNumberingBuilder.trackerNumberingGeometry_cfi")
+process.load("Geometry.HcalCommonData.hcalParameters_cfi")
```

Comment:
@friccita @ianna - here and elsewhere in this pull request - could we not just use the full geometry cffs already defined instead of try to reformulate them here? It would be easier to maintain.